src/main/java/woowahan/anifarm/tecolearning/user/controller/UserController.java

```diff
@@ -0,0 +1,35 @@
+package woowahan.anifarm.tecolearning.user.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import woowahan.anifarm.tecolearning.user.dto.UserCreateDto;
+import woowahan.anifarm.tecolearning.user.dto.UserInfoDto;
+import woowahan.anifarm.tecolearning.user.dto.UserUpdateDto;
+import woowahan.anifarm.tecolearning.user.service.UserService;
+
+@RestController
+@RequestMapping("/api/users")
+public class UserController {
+    private UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @PostMapping("/signup")
+    public ResponseEntity create(@RequestBody UserCreateDto userCreateDto) {
+        userService.save(userCreateDto);
+        return ResponseEntity.ok().build();
+    }
+
+    @GetMapping("/{userId}")
+    public ResponseEntity<UserInfoDto> read(@PathVariable long userId) {
```

Comment:
```suggestion
    public ResponseEntity<UserInfoDto> read(@PathVariable Long userId) {
```
Entity의 경우 wrapper type을 사용하는데 이 부분은 어떡하는 것이 좋을까요?

Reply:
Entity만 wrapper 사용 나머지는 기본 타입으로 사용합시다!